README.md

```diff
@@ -83,6 +85,8 @@ gh auth login
 gh config set git_protocol=ssh
 ```
 
+<!-- Is the tracked `purs` binary cross-platform? What about adding a `wget`/`curl` step to `setup.sh` and removing the tracked binary? Also, might want to do a squash/force-push to remove the 50MB binary from git history. It takes a while to clone the repo on slow connections otherwise. -->
+
```

Comment:
Wow... Duh! We'll use the same approach that the purescript repos use in their `.travis.yml` file.